### PR TITLE
Comp sim

### PR DESCRIPTION
**Pull request by Joe-Joe-Joe-Joe**
- Cleaned up old commented-out code
- Modified controller logic to only run the controller in coast phase of flight (prevents windup)
- Modified daemon task priority so that freeRTOS functions can actually run from ISRs
- Added a couple more logs in places

### CODE REVIEW

File: STM32Cube/Core/Src/main.c [hunk 1]
```diff
@@ -235,9 +234,9 @@ int main(void)
   xReturned &= xTaskCreate(trajectory_task, "traj", 512, NULL, (UBaseType_t) osPriorityNormal, &trajectoryTaskHandle);
   xReturned &= xTaskCreate(logTask, "Logging", 1024, NULL, (UBaseType_t) osPriorityNormal, &logTaskhandle);
   xReturned &= xTaskCreate(healthCheckTask, "health checks", 512, NULL, (UBaseType_t) osPriorityNormal, &healthChecksTaskHandle);
-  xReturned &= xTaskCreate(controlTask, "Controller", 512, NULL, (UBaseType_t) osPriorityBelowNormal, &controllerHandle);
-  xReturned &= xTaskCreate(flightPhaseTask, "Flight Phase", 512, NULL, (UBaseType_t) osPriorityAboveNormal, &flightPhaseHandle);
-#ifdef TEST_MODE
+  xReturned &= xTaskCreate(controlTask, "Controller", 512, NULL, (UBaseType_t) osPriorityNormal, &controllerHandle);
+  xReturned &= xTaskCreate(flightPhaseTask, "Flight Phase", 512, NULL, (UBaseType_t) osPriorityNormal, &flightPhaseHandle);
+#if  defined(TEST_MODE) && defined(DEBUG)
```
Comment (Joe-Joe-Joe-Joe) [inline STM32Cube/Core/Src/main.c:239]: Stops us from accidentally running otits in prod (though its also a define so I guess it should never run in prod config anyway) :face:

File: STM32Cube/Tasks/state_estimation.c [hunk 2]
```diff
@@ -185,9 +185,9 @@ void stateEstTask(void *arguments) {
 			xQueueOverwrite(angleQueue, &euler);
 
 			can_msg_t msg;
-			if(build_state_est_data_msg(millis_(), &euler.angle.roll, STATE_ANGLE_ROLL, &msg)) xQueueSend(busQueue, &msg, 10);
-			if(build_state_est_data_msg(millis_(), &euler.angle.pitch, STATE_ANGLE_PITCH, &msg)) xQueueSend(busQueue, &msg, 10);
-			if(build_state_est_data_msg(millis_(), &euler.angle.yaw, STATE_ANGLE_YAW, &msg)) xQueueSend(busQueue, &msg, 10);
+			if(build_state_est_data_msg(millis_(), &euler.angle.roll, STATE_FILTER_ROLL, &msg)) xQueueSend(busQueue, &msg, 10);
```
Comment (Joe-Joe-Joe-Joe) [inline STM32Cube/Tasks/state_estimation.c:188]: this somehow got reverted on master?

**Review by celery6 — CHANGES_REQUESTED** [review]

File: STM32Cube/Tasks/controller.c [hunk 3]
```diff
@@ -69,15 +73,13 @@ void controlTask(void *argument)
...
-			{
-				can_msg_t msg;
-				build_actuator_cmd_analog( (uint32_t) millis_(), ACTUATOR_AIRBRAKES_SERVO, (uint8_t) (100 * extension), &msg);
-				xQueueSend(busQueue, &msg, 5); //If we are in the coast phase, command the airbrakes servo to the target extension value
-			}
+			can_msg_t msg;
+			build_actuator_cmd_analog( (uint32_t) millis_(), ACTUATOR_AIRBRAKES_SERVO, cmd_extension, &msg);
+			xQueueSend(busQueue, &msg, 5); //If we are in the coast phase, command the airbrakes servo to the target extension value
+			logInfo("controller", "ext CMD: %d", cmd_extension);
+			printf_("extension: %f\n", extension);
```
[the remark this answers is not in the document]
Comment (Joe-Joe-Joe-Joe) [reply STM32Cube/Tasks/controller.c:82]: okay but why
the whole point is that it gets compiled out in prod

[the remark this answers is not in the document]
Comment (celery6) [reply STM32Cube/Tasks/controller.c:82]: does it get compiled out??? there's a ifdef around putchar but nothing guarding the actual printf which is annoyingly large computing sink

[the remark this answers is not in the document]
Comment (Joe-Joe-Joe-Joe) [reply STM32Cube/Tasks/controller.c:82]: ah okay I will comment them out for now and we can revisit after tomorrow

**Review by celery6 — APPROVED** [review]

**Pull request merged**